Recipes/JsonConfiguration/CompactJsonSerializer.cs

```diff
@@ -0,0 +1,88 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="CompactJsonSerializer.cs">
+//   Copyright (c) 2016. All rights reserved.
+//   Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+// <auto-generated>
+//   Sourced from NuGet package. Will be overwritten with package update except in Spritely.Recipes source.
+// </auto-generated>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Spritely.Recipes
+{
+    using System;
+
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// This is a wrapper around <see cref="JsonConvert"/> that uses the 
+    /// serialization settings from <see cref="JsonConfiguration.CompactSerializerSettings"/>.
+    /// This is for internal use-cases where these settings are specifically required 
+    /// and where the caller does not want to rely on <see cref="JsonConvert.DefaultSettings"/>
+    /// </summary>
+#if !RecipesProject
+    [System.Diagnostics.DebuggerStepThrough]
+    [System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
+    [System.CodeDom.Compiler.GeneratedCode("Spritely.Recipes", "See package version number")]
+#endif
+    internal static partial class CompactJsonSerializer
+    {
+        /// <summary>
+        /// Deserializes the JSON to the specified .NET type.
+        /// </summary>
+        /// <typeparam name="T">The type of the object to deserialize to.</typeparam>
+        /// <param name="value">The JSON to deserialize.</param>
+        /// <returns>The deserialized object from the JSON string.</returns>
+        public static T DeserializeObject<T>(string value) where T : class
```

Comment:
I liked the terser name too, but if you look at the list of JsonConvert methods, you'll see DeserializeAnonymousType.  So if we add that later, I thought I might be confusing so I made it consistent with JsonConvert and used DeserializeObject.  Same issue with Serialize (there's a SerializeXmlNode and others)

http://www.newtonsoft.com/json/help/html/Methods_T_Newtonsoft_Json_JsonConvert.htm